### DISCUSSION TITLE
Admin Password Reset

### DISCUSSION THREAD

**Discussion by MaoMaoCake** [dani-garcia/vaultwarden#3864] [Q&A]
The [wiki](https://github.com/dani-garcia/vaultwarden/wiki) shows support for the feature however I cannot find anywhere in the UI. Is this a feature that needs to be enabled somewhere?  The only things I can find is someone implemented it.

**Comment by BlackDex** [comment]
Read: https://bitwarden.com/help/account-recovery/

**Comment by MaoMaoCake** [comment]
Here is the support string
```
### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.29.2
* Web-vault version: v2023.7.1
* OS/Arch: linux/x86_64
* Running within Docker: true (Base: Debian)
* Environment settings overridden: false
* Uses a reverse proxy: true
* IP Header check: false (X-Forwarded-For)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Browser/Server Time Check: true
* Server/NTP Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.41.2
* Clients used: 
* Reverse proxy and version: 
* Other relevant information: 

### Config (Generated via diagnostics page)
<details><summary>Show Running Config</summary>
```

here is what I see
![Screenshot from 2023-09-07 17-37-11](https://github.com/dani-garcia/vaultwarden/assets/55589246/311234cd-3303-4929-bbee-7a2e06b42455)

and this is what I expect to see
![Screenshot from 2023-09-07 17-36-52](https://github.com/dani-garcia/vaultwarden/assets/55589246/c5eeae97-31c6-467e-b9c4-bfe3c5b98873)

I don't see how to activate it in VW

**Reply by BlackDex** [reply]
Then you do not have SMTP/Mail enabled, that is a mandatory requirement.
